js/views/standalone_registration_view.js

```diff
@@ -8,6 +8,10 @@
 
   window.Whisper = window.Whisper || {};
 
+  const registerIndex = 0;
+  const profileIndex = 1;
```

Comment:
constants as uppercase?

Reply:
Deliberately left in there so you could try out the multi line comments

Reply:
You're too kind